docs/fluvio/concepts/architecture/replica-assignment.mdx

```diff
@@ -7,14 +7,10 @@ title: "Replica Assignment"
 
 For additional information on replica sets, checkout [replica election].
 
-[replica election]:
-
 ## Replica Assignment Algorithm
 
 Fluvio replica assignment algorithm ensures that the replica leader and followers are evenly distributed across available SPUs. If you'd rather deploy your own replication algorithm, use [Manual Replica Assignment] instead.
 
-[Manual Replica Assignment]:
```

Comment:
Where is this one defined?

Reply:
This is meant to refer to an H3 on the same page. How do I do that?

Reply:
found it